assets/less/bootstrap/code.less

```diff
@@ -57,5 +57,5 @@ pre {
 // Enable scrollable blocks of code
 .pre-scrollable {
   max-height: 340px;
-  overflow-y: scroll;
+  overflow-y: auto;
 }
```

Comment:
hey we're not using this class anywhere in our code, i'm just gonna take out my mod. if we start using the bootstrap class, lets just fix it when we cross that bridge